doc/snippets/Microsoft.Data.SqlClient/SqlBatchCommandCollection.xml

```diff
@@ -146,31 +135,46 @@
       <param name="index">
         The index where the <see cref="T:Microsoft.Data.SqlClient.SqlBatchCommand" /> object should be located.
       </param>
-      <param name="batchCommand">To be added.</param>
       <summary>
```

Comment:
Build break here:
```suggestion
      <param name="batchCommand">
          The <see cref="T:Microsoft.Data.SqlClient.SqlBatchCommand" /> object.
      </param>
      <summary>
```

Reply:
Thanks for that - I've fixed in the most recent commit.